saleor/webhook/payloads.py

```diff
@@ -93,7 +93,13 @@ def prepare_order_lines_allocations_payload(line):
     return warehouse_id_quantity_allocated_map
 
 
+def _charge_taxes(order_line: OrderLine) -> Optional[bool]:
+    variant = order_line.variant
+    return None if not variant else variant.product.charge_taxes
```

Comment:
Just to make sure - this is a deliberate decision to have here a `None` or a boolean value? Wouldn't true/false be enough?

Reply:
Yes, without it external app couldn't determine whether the variant exists for the order line.